webhooks-extension/pkg/endpoints/webhook.go

```diff
@@ -1080,6 +1083,7 @@ func (r Resource) deleteFromEventListener(name, installNS, monitorTriggerNamePre
 		}
 	}
 
+	logging.Log.Debugf("Returning: %s", err)
```

Comment:
Is it worth mentioning the method name here in this debugf?

Reply:
This was really just a debug for me, now gone